install_test_visibility.sh

```diff
@@ -16,6 +16,14 @@ if ! mkdir -p $ARTIFACTS_FOLDER; then
   return 1
 fi
 
+extract_major_version() {
+  echo $1 | cut -d '.' -f 1
```

Comment:
I recommend always wrapping shell variables in double quotes, unless you actually want the shell to do word splitting. (Although I realize that this script already has other unquoted variables not introduced by this PR.)
```suggestion
  echo "$1" | cut -d '.' -f 1
```

Reply:
Fixed in https://github.com/DataDog/test-visibility-install-script/pull/12/commits/31cb0fa013f03cae902a555b52cf5ed378341775